man/systemd.resource-control.xml

```diff
@@ -214,6 +214,42 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>CPUSetCpus</varname></term>
```

Comment:
With the naming I was following other cgroups settings and basically all of them copy the kernel file naming but I'm OK with the suggested names.  For the effective I would go with "EffectiveCPUs=" and "EffectiveMemoryNodes=", does that work?

Reply:
True. But so far the names we got this way sounded OK. E.g. CPUWeight — if I had complete freedom to pick the name for this, I'd probably still pick that. But in case of cpuset, suddenly the implementation detail of this being a separate controller leaks into the user-visible naming very prominently.

>  For the effective I would go with "EffectiveCPUs=" and "EffectiveMemoryNodes=", does that work?

Yep. This, or maybe CPUsEffective= and MemoryNodesEffective= (to make sort order nicer)? I think either is OK.

Reply:
I probably prefer EffectiveCPUs to be aligned with AllowedCPUs, thanks.